go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/stretchr/testify v1.8.2
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.25.1-0.20230511132046-5e380c8d14e7
+	go.mongodb.org/atlas v0.25.1-0.20230515165503-b942fd340c0d
```

Comment:
we have recently released v0.26. Is there any reason why this is still v0.25? You may want to merge master into the SDK v2 to get the latest fixes

Reply:
Yep prob outdated sdkv2. I'll update.

Thanks for the review 🙇